embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/opentelemetry/OpenTelemetrySdk.kt

```diff
@@ -16,11 +17,17 @@ internal class OpenTelemetrySdk(
     openTelemetryClock: Clock,
     spanProcessor: SpanProcessor
 ) {
+    private var resource: Resource = Resource.getDefault().toBuilder()
+        .put("service.name", BuildConfig.LIBRARY_PACKAGE_NAME)
```

Comment:
Also, this is typically the name of the app. It's not something we capture is it? This is fine for now, but we might want to allow customers to specify an app name - or we default to the appId or something. Same thing for appVersion.